content/references/rippled-api/public-rippled-methods/ledger-methods/ledger_entry.md

```diff
@@ -38,6 +36,12 @@ An example of the request format:
 }
 ```
 
+*Commandline*
+
+```
+rippled json ledger_entry '{ "account_root": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59", "ledger_index": "validated", "type": "account_root" }'
```

Comment:
For consistency, if we're going to provide examples of the `json` method workaround for the methods that don't have native commandline syntax, we should do that on the other methods above too